projects/swimlane/ngx-charts/src/lib/number-card/number-card.component.ts

```diff
@@ -70,15 +72,7 @@ export class NumberCardComponent extends BaseChartComponent {
     this.setColors();
     this.transform = `translate(${this.dims.xOffset} , ${this.margin[0]})`;
 
-    const size = gridSize(this.dims, this.results.length, 150);
-    const N = size[0] * size[1];
-
     const data = this.results.slice();
-
-    while (data.length < N) {
-      data.push({ value: null });
-    }
```

Comment:
When enforcing the typings we are not allow to push "empty" items.

AFAIK this "filling up" is to have empty cells. Now, I am wondering if these are really needed or not? If so, I think I'd simply revert these changes. WDYT?

|Before|After|
|-|-|
|![image](https://user-images.githubusercontent.com/12933406/125645424-f0ada5f3-7c41-4ece-888e-1fbe2343464f.png)|![image](https://user-images.githubusercontent.com/12933406/125645457-9dfd5f9a-1ce9-4e7a-ad1f-9f47fe7805c7.png)|

source: https://swimlane.github.io/ngx-charts/#/ngx-charts/status-demo